documentation/src/main/asciidoc/topics/ref_rest_logging.adoc

```diff
@@ -0,0 +1,57 @@
+[id='rest_v2_logging_list_appenders']
+= Listing the logging appenders
+View a list of all configured appenders with `GET` requests.
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /rest/v2/logging/appenders
```

Comment:
It should be documented that it will only get the logger from the local server, or move the API to under /server